app/bundles/EmailBundle/Views/SubscribedEvents/Timeline/index.html.twig

```diff
@@ -0,0 +1,68 @@
+{#
+    @copyright   2014 Mautic Contributors. All rights reserved
+    @author      Mautic
+    @link        http://mautic.org
+    @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+#}
+
+{% if (event.extra is defined) ? event.extra.stat : false %}
+    {% set item = event.extra %}
+<p>
+    {% if item.isFailed is not empty %}
+
+        {% if item.openDetails.bounces is defined %}
+            <span class="label label-warning" data-toggle="tooltip" title="{% trans %}mautic.email.timeline.event.bounced{% endtrans %}">
+                        {% trans %}mautic.email.timeline.event.bounced{% endtrans %}
+            </span>
+        {% else %}
+            {% trans %}mautic.email.timeline.event.failed{% endtrans %}
+        {% endif %}
+    {% elseif item.dateRead is not empty %}
+        {% trans %}mautic.email.timeline.event.not.read{% endtrans %}
+    {% else %}
+        {{ 'mautic.email.timeline.event.' ~ event.extra.type|trans ({
+            '%date%'     : dateToFull(event.timestamp),
+            '%interval%' : dateToDate(event.timestamp),
+            '%sent%'     : dateToFull(item.dateSent),
+        }) }}
+    {% endif %}
+    {% if item.viewedInBrowser is not empty %}
+        {% trans %}mautic.email.timeline.event.viewed.in.browser{% endtrans %}
+    {% endif %}
+    {% if item.retryCount is not empty %}
+        {{ 'mautic.email.timeline.event.retried'|trans({'%count%' : item.retryCount}) }}
+    {% endif %}
+    {% if item.list_name is not empty %}
+        {{ 'mautic.email.timeline.event.list'|trans({'%list%' : item.list_name}) }}
+    {% endif %}
+</p>
+<div class="small">
+    {% if item.openDetails.bounces is defined %}
+        {% set item.openDetails = item.openDetails|filter(v, k) => k != 'bounces' %}
+    {% endif %}
+
+    {% if item.openDetails is not empty %}
+        <h6 class="mt-lg mb-sm"><strong>{% trans %}mautic.email.timeline.open_details{% endtrans %}</strong></h6>
+    {% endif %}
+
+    {% set counter = 1 %}
+    {% for detail in item.openDetails %}
+        {% if showMore is empty and counter > 5 %}
+            {% set showMore = true %}
+            {{ '<div style="display:none">' }}
+        {% endif %}
+        {% if counter > 1 %}
+            {{ '<hr/>' }}
+        {% endif %}
+        <strong>{{ dateToText(detail.datetime, 'UTC') }}</strong><br/>{{ detail.useragent|escape }}
+        {% set counter = counter + 1 %}
+    {% endfor %}
+
+    {% if showMore is not empty %}
+        {{ '</div>' }}
+        {{ '<a href="javascript:void(0);" class="text-center small center-block mt-xs" onclick="Mautic.toggleTimelineMoreVisiblity(mQuery(this).prev());">' }}
+        {{ mautic.core.more.show|trans }}
+        {{ '</a>' }}
```

Comment:
I can see that this was done the same way in the original file, but can the HTML be there without echoing it as a string? Same in other places in this file.